src/main/java/de/propra/timetracker/TablePrinter.java

```diff
@@ -1,25 +1,49 @@
 package de.propra.timetracker;
 
 import net.steppschuh.markdowngenerator.table.Table;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
 import java.util.List;
 
 public class TablePrinter {
 
     static void printTable(List<Event> events) {
-        Table.Builder tableBuilder = new Table.Builder()
-                .withAlignments(Table.ALIGN_LEFT, Table.ALIGN_LEFT, Table.ALIGN_LEFT, Table.ALIGN_LEFT)
-                .addRow("Datum", "Minuten", "Projekt", "Beschreibung");
+        Table.Builder tableBuilder = header();
 
         events.forEach(e -> tableBuilder.addRow(e.datum(),e.minuten(),e.projekt(),e.beschreibung()));
         System.out.println(tableBuilder.build());
     }
 
-    static void printTableOf(List<Event> events, String projekt) {
-        Table.Builder tableBuilder = new Table.Builder()
-                .withAlignments(Table.ALIGN_LEFT, Table.ALIGN_LEFT, Table.ALIGN_LEFT, Table.ALIGN_LEFT)
-                .addRow("Datum", "Minuten", "Projekt", "Beschreibung");
+    static void printTableOfProject(List<Event> events, String projekt) {
+        Table.Builder tableBuilder = header();
+        events.stream().filter(e -> e.projekt().equals(projekt))
+                .forEach(en -> tableBuilder.addRow(en.datum(),en.minuten(),en.projekt(),en.beschreibung()));
+        System.out.println(tableBuilder.build());
+    }
 
-        events.stream().filter(e -> e.projekt().equals(projekt)).forEach(e -> tableBuilder.addRow(e.datum(),e.minuten(),e.projekt(),e.beschreibung()));
+    static void printTableOfDate(List<Event> events, String datum) {
+        Table.Builder tableBuilder = header();
+        SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd");
+        Date date;
+        String dateString;
+        try {
+            date = format.parse(datum);
+            dateString = format.format(date);
+        } catch (ParseException e) {
+            throw new RuntimeException(e);
+        }
+        events.stream().filter(d -> d.projekt().equals(dateString))
+                .forEach(d -> tableBuilder
+                        .addRow(d.datum(),d.minuten(),d.projekt(),d.beschreibung()));
         System.out.println(tableBuilder.build());
     }
+
+    static Table.Builder header() {
+        return new Table.Builder()
+                .withAlignments(Table.ALIGN_LEFT, Table.ALIGN_LEFT, Table.ALIGN_LEFT, Table.ALIGN_LEFT)
+                .addRow((Object[]) Spalten.values());
+    }
```

Comment:
das hier bitte als statisches Feld in der Klasse speichern, so wie hier: https://github.com/hhu-propra1/time-tracker/blob/main/src/main/java/de/propra/timetracker/CSV.java#L17-L20